README.md

```diff
@@ -69,3 +69,10 @@ srclib by installing an editor plugin in your editor of choice. [See all editor
 ## License
 Sourcegraph is licensed under the [MIT License](https://tldrlegal.com/license/mit-license).
 More information in the LICENSE file.
+
+
+## Contributing
+
+**In case you want to get involved and start hacking on srclib bugs and features or write your own toolchain, [sign up for Slack](http://slackin.srclib.org/) and access it on  [srclib.slack.com](http://srclib.slack.com) .**
```

Comment:
@jelmerdereus I think @shurcooL was saying change the text to be something like:

```
If you want to start hacking on srclib or write your own srclib toolchain, [join the srclib Slack](slackin.srclib.org) and then access it on [srclib.slack.com](http://srclib.slack.com).
```